yt/yt/ytlib/query_tracker_client/query_tracker_service_proxy.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include <yt/yt/ytlib/query_tracker_client/proto/query_tracker_service.pb.h>
+
+#include <yt/yt/core/rpc/client.h>
+
+namespace NYT::NQueryTrackerClient {
+
+////////////////////////////////////////////////////////////////////////////////
+
+class TQueryTrackerServiceProxy
+    : public NRpc::TProxyBase
+{
+public:
+    DEFINE_RPC_PROXY(TQueryTrackerServiceProxy, TQueryTrackerService,
+        .SetProtocolVersion(NRpc::TProtocolVersion{0, 0}));
+
+    DEFINE_RPC_PROXY_METHOD(NProto, StartQuery);
```

Comment:
Is it possible to remove code duplication here too? I think that having multiple places with the same RPC_PROXY_METHOD definitions may be confusing. Do we have a plan for the future refactoring of the Query Tracker RPC Proxy methods?
https://github.com/ytsaurus/ytsaurus/blob/main/yt/yt/client/api/rpc_proxy/api_service_proxy.h#L187

Reply:
I don't understand how we can remove this. I would appreciate any advice on how to do it.

There are no plans for refactoring, but it would be great if you could describe how it could be done better.

Reply:
I realized that the protobuf types are different. I think it's fine in this case.